File: 240904/두 사람/two-person.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int a_age = sc.nextInt();
        String a_gender = sc.next();

        int b_age = sc.nextInt();
        String b_gender = sc.next();

        String man = "M";

        if(((a_age >= 19) || ( b_age >= 19)) && ((a_gender.equals(man)) || (b_gender.equals(man)))){
            System.out.print("1");
        } else {
            System.out.print("0");
        }
    }
}

File: 250311/두 사람/two-person.js
const fs = require("fs");

let arr = fs.readFileSync(0).toString().split(/ |\n/);

let aAge = Number(arr[0]), aGender = arr[1];
let bAge = Number(arr[2]), bGender = arr[3];

if((aAge >= 19 && aGender === "M") || (bAge >= 19 && bGender === "M")){
    console.log(1);
} else {
    console.log(0);
}